rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -2396,3 +2396,35 @@ rocalNop(RocalContext p_context,
     }
     return output;
 }
+
+RocalTensor ROCAL_API_CALL
+rocalPreEmphasisFilter(RocalContext p_context,
+                       RocalTensor p_input,
+                       RocalTensorOutputType rocal_tensor_output_datatype,
+                       bool is_output,
+                       RocalFloatParam p_preemph_coeff,
+                       RocalAudioBorderType preemph_border_type) {
+    if(!p_context || !p_input)
+        THROW("Null values passed as input")
+    Tensor* output = nullptr;
```

Comment:
Error handling with throw statement has been slightly modified in latest image api calls
` Tensor* output = nullptr;
    if ((p_context == nullptr) || (p_input == nullptr)) {
        ERR("Invalid ROCAL context or invalid input tensor")
        return output;
    }`

Please check and change here as well appropriately

Reply:
done